### DISCUSSION TITLE
Support for Gluetun for Homepage in Synology

### DISCUSSION THREAD

**Discussion by LETRA** [qdm12/gluetun#1287] [Q&A] [answered]
Hi. I have a Gluetun installation with docker on Synology and I need to enable the log and change the port https://github.com/qdm12/gluetun/wiki/HTTP-control-server-options to display the data with Homepage https://github.com/benphelps/homepage/pull/512 I need change this parameters HTTP_CONTROL_SERVER_ADDRESS & HTTP_CONTROL_SERVER_LOG How can I do it?

**Comment by Tarow** [comment]
As mentioned in the linked PR:
The HTTP Control Server is already running (default port 8000). All you have to do is expose it to the host system. So in your docker-compose file, add `8000:8000` under the `ports` section.

Then in your homepage-widget you have to specify the url including the port like `http://ip.of.your.server:8000`.

**Reply by qdm12** [reply]
Just dropping a thanks for helping :heart:

**Reply by Arelius-D** [reply]
Guys, could you please throw me a bone here I was [over here](https://github.com/gethomepage/homepage/discussions/639) and then thought maybe I rather ask around here. @qdm12 / @Tarow is it possible for any on you to shed some light onto this for a novice like myself? I don't understand in which container the suggest port need to be added..

**Reply by Arelius-D** [reply]
Never mind my request guys, got help and my issue is solved [click for details](https://github.com/gethomepage/homepage/discussions/639).

**Comment by LETRA** [comment]
It doesn't work with port 8000. It's a very common port and I probably have another service there that it's conflicting with. What exactly would be the syntax to add those parameter to the docker-compose file? This for example?

HTTP_CONTROL_SERVER_ADDRESS: 3853

**Comment by Tarow** [comment, marked as answer]
You might just want to leave it at default and map it to another host port. Like `3853:8000`.

If you want to set the environment variable, u would do it like:
```
environment:
  - HTTP_CONTROL_SERVER_ADDRESS=:3853
      
```
in your docker-compose file.
